backend/app/models.py

```
@@ -712,3 +713,160 @@ class Meta:
+    """
+    STATUS_CHOICES = [
+        ("analyzing", "Analyzing"),
+        ("dry_run", "Dry Run"),
+        ("pending", "Pending"),
+        ("running", "Running"),
+        ("completed", "Completed"),
+        ("completed_errors", "Completed with errors"),
+        ("failed", "Failed"),
+        ("reverting", "Reverting"),
+        ("reverted", "Reverted"),
+    ]
```

Ruff (0.15.21)

[warning] Mutable default value for class attribute

(RUF012)